spec/derivative_services/audio_derivative_service_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+require "rails_helper"
+require "valkyrie/derivatives/specs/shared_specs"
+include ActionDispatch::TestProcess
+
+RSpec.describe AudioDerivativeService do
+  it_behaves_like "a Valkyrie::Derivatives::DerivativeService"
+  let(:derivative_service) do
+    AudioDerivativeService::Factory.new(change_set_persister: change_set_persister)
+  end
+  let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+  let(:storage_adapter) { Valkyrie.config.storage_adapter }
+  let(:persister) { adapter.persister }
+  let(:query_service) { adapter.query_service }
+  let(:file) { fixture_file_upload("files/audio_file.wav", "audio/x-wav") }
```

Comment:
@tpendragon how did you get this mime type to work without using `run_real_characterization: true`?